R/data.R

```diff
@@ -504,6 +504,14 @@
 #'
 "delta_total_diverted"
 
+#' @title Flow at Wilkins
+#' @description flow in cms at Wilkins (C126)
+#' @format A matrix with months (1-12) as rows and years (1980-1999) as columns
+#' @details
+#' The flows here are obtained from a CALSIM run designed for the structured
+#' decision making in the delta. The name of the scenario is: Full Obs No Notch Baseline.
```

Comment:
the `Full Obs No Notch Baseline` was used for the delta sdm, and was brought over to this model as part of the updated delta routing, but it looks like this calsim data is no longer from `FullObsJul18_NoNotch_Base_DV_filtered.csv` and is instead obtained from a recent dss file, is this technically still the same scenario? Can you check to DSS to see if there are details or a name for the scenario

Reply:
Ah yes let me check that.

Reply:
this is actually about the `cvp_exports` and `swp_exports` but it will affect all datasets that use to be derived from `FullObsJul18_NoNotch_Base_DV_filtered`

Reply:
Yes. I will fix all documentation if it is a different scenario.